Extension/package.json

```diff
@@ -495,6 +495,18 @@
           "default": true,
           "description": "If true, snippets are provided by the language server.",
           "scope": "resource"
+        },
+        "C_Cpp.textMateColorization": {
```

Comment:
I don't think we should officially ship this setting.  You may still be able to use it in the extension for testing without declaring it here though...  There is code that iterates through the settings to see what's changed, and in that case we actually see everything in the JSON object representing the C_Cpp section whether it's declared in package.json or not.  You could potentially do something similar for this one.

Reply:
Are you sure we shouldn't ship with this setting?  The two systems are significantly different.  That's not apparent with the few color themes we've looked at, as very little that differs is colored.  However, if someone takes advantage of the ability to color the various items we allow to be colored, the output can be significantly different, resulting in scenarios where the color swapping is apparent and distracting.  (i.e. on making a file active - first textMate colors are applied, then ours).

Reply:
As discussed, kept the setting but not publishing it in the schema.